modules/rxrtbBidAdapter.js

```diff
@@ -100,7 +100,7 @@ function makeImp(req) {
     'banner': makeBanner(req)
   };
 
-  if (req.params.bidfloor && utils.isInteger(req.params.bidfloor)) {
+  if (req.params.bidfloor && Number.isFinite(req.params.bidfloor)) {
```

Comment:
We don't support all the ES6 features (for example, IE 11 doesn't support this). I'd suggest you use `isFinite` global if you want to check against this.